src/missing-types.d.ts

```diff
@@ -565,6 +566,15 @@ declare module '@higlass/tracks' {
         exportSVG(): [HTMLElement, HTMLElement];
     }
 
+    export class SVGTrack<Options> extends Track<Options> {
+        /* Properties */
+        gMain: d3Selection.Selection<d3Selection.Datum, d3Selection.PElement, d3Selection.PDatum>;
+        clipUid: string;
+        clipRect: d3Selection.Selection<d3Selection.Datum, d3Selection.PElement, d3Selection.PDatum>;
+        /* Constructor */
+        constructor(context: Context<unknown, Options>, options: Options);
+    }
+
```

Comment:
Corresponds to [SVGTrack](https://github.com/higlass/higlass/blob/b2ee5940c519982dc53685153ff863d64443d0bb/app/scripts/SVGTrack.js#L6) in HiGlass